docs/guides/hedgehog-engine/blueblur/files/xncp/anims.md

```diff
@@ -39,19 +39,19 @@ So, in Intro_Anim, we have several **CastAnimationData**. These control what hap
 
 Seems like we have flags! It’s a bitfield, and it controls what actually animates in this cast. Here’s the meaning of each bit:
 ```
-0000 0000 - Nothing
-0000 0001 - Unknown
-0000 0010 - X Position
-0000 0100 - Y Position
-0000 1000 - Rotation
-0001 0000 - X Scale
-0010 0000 - Y Scale
-0100 0000 - SubImage
-1000 0000 - Colour Mask
+0000 0000(0) - Nothing
+0000 0001(1) - Hide Flag
+0000 0010(2) - X Position
+0000 0100(4) - Y Position
+0000 1000(8) - Rotation
+0001 0000(16) - X Scale
+0010 0000(32) - Y Scale
+0100 0000(64) - SubImage
+1000 0000(128) - Colour Mask
 ```
 
-So, with this info, we now know what animates in the position cast: It’s its X Position.
-The **CastAnimationDataSubData1List** is what stores the actual animation data. The amount of entries in this list depends on the number of bits signaled in the **Flags** field. The first entry in the list is referring to the first bit signaled from right to left. So by that logic, the list only has one entry.
+So, with this info, we now know what animates in the position cast: It is its X Position.
+The **CastAnimationDataSubData1List** is what stores the actual animation data. The amount of entries in this list depends on the number of bits *(or combination of decimal numbers)* signaled in the **Flags** field. The first entry in the list is referring to the first bit signaled from right to left. So by that logic, the list only has one entry.

```

Comment:
I feel like "number of signaled bits" is enough here, as that's really what controls the amount of entries in the subdata list. Decimal numbers don't seem to make much sense in this context.

Reply:
The thing is that again, most people by default will probably have decimal view in 010, and flags there are just type1+type2+type3 etc in animations. So, i thought it would fit considering it makes it easier for the average guy to understand it better

Reply:
Hmmm ok, I understand that. However, I feel like if someone actually wants to edit these kinds of files manually, they would need to have an understanding of what "bits" are, which isn't something hard to learn.